guides/common/assembly_preparing-environment-for-installation.adoc

```diff
@@ -3,7 +3,11 @@ ifdef::context[:parent-context: {context}]
 [id="Preparing_your_Environment_for_Installation_{context}"]
 = Preparing your environment for installation
 
-Before you install {Project}, ensure that your environment meets the following requirements.
+Review prerequisites below before you install {ProjectServer}.
```

Comment:
Also based on https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/con_upgrading-overview.adoc for consistency